Userland/Utilities/image.cpp

```diff
@@ -109,6 +112,9 @@ static ErrorOr<void> strip_alpha(LoadedImage& image)
     case Gfx::BitmapFormat::BGRA8888:
     case Gfx::BitmapFormat::BGRx8888:
         frame->strip_alpha_channel();
+        break;
+    default:
+        VERIFY_NOT_REACHED();
```

Comment:
Here (and on line 92) you add default cases to the switches. Please move the VERIFY_NOT_REACHED() to underneath the switch block and remove the `default:` case. Instead, implement the newly added enumeration value - even if it means adding a `FIXME`.

This way, the compiler will let you know whenever you add a new enumeration value and forget to expand the switch cases.

Reply:
I didn't know compiler checks that. As it does, we really don't need any VERIFY_NOT_REACHED here. I've fixed that